data_import/forms.py

```diff
@@ -40,3 +46,10 @@ def clean_reporting_year(self):
             raise forms.ValidationError('Reporting year cannot exceed the current year')
 
         return reporting_year
+
+    def form_valid(self, form):
+        uploaded_file = form.cleaned_data['standardized_file']
```

Comment:
Thinking more about this: Is there a reason to do this here, instead of at the end of `clean_standardized_file`?